deploy-gatsby.md

```diff
@@ -0,0 +1,28 @@
+# Deploying Gatsby
+
+## Best Practice
+
+
+Though you can deploy from the same location multiple times it is recommended that you clear your public directory before each build
```

Comment:
This is *not* the recommended practice actually. The only thing that needs deleted between builds is html files of pages that don't exist anymore.

You actually want to keep old builds around because when deploying often, people with your site open can still pull JS/Assets from their particular version of your site.